README.md

```diff
@@ -280,6 +280,49 @@ Please go the Remote Management tab of the Settings page and check which remote
 
 ![Remote Management Tab](./assets/screenshot-3.png)
 
+### RESTapi for Remote Management
```

Comment:
Is the standard `RESTap` or is it `REST API`? We should try to be consistent with how WP uses the expression.